analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -88,7 +88,7 @@ class NuGetSupport(serviceIndexUrls: List<String> = listOf(DEFAULT_SERVICE_INDEX
             val configFile = definitionFile.parentFile.searchUpwardsForFile("nuget.config", ignoreCase = true)
             val serviceIndexUrls = configFile?.let { configXmlReader.getRegistrationsBaseUrls(it) }
 
-            return NuGetSupport(serviceIndexUrls.orEmpty())
+            return NuGetSupport(serviceIndexUrls ?: listOf(DEFAULT_SERVICE_INDEX_URL))
```

Comment:
`listOf(DEFAULT_SERVICE_INDEX_URL)` is already the default: `class NuGetSupport(serviceIndexUrls: List<String> = listOf(DEFAULT_SERVICE_INDEX_URL)) {`

This should either be used in this case like `return serviceIndexUrls?.let { NuGetSupport(it) } ?: NuGetSupport()` or the default in the constructor should be removed.